scripts/ci/docker_upgrade.sh

```diff
@@ -61,5 +61,5 @@ docker push "${DOCKER_IMAGE_ID}-${VERSION}"
 
 REPO_DIGEST=$(docker inspect --format='{{.RepoDigests}}' "${DOCKER_IMAGE_ID}-${VERSION}")
 
-echo "::set-env name=DOCKER_IMAGE::${DOCKER_IMAGE_ID}-${VERSION}"
-echo "::set-env name=DOCKER_REPO_DIGEST::${REPO_DIGEST}"
+echo "DOCKER_IMAGE=${DOCKER_IMAGE_ID}-${VERSION}" >> $GITHUB_ENV
```

Comment:
Due to a [security vulnerability](https://github.com/actions/toolkit/security/advisories/GHSA-mfwh-5m23-j46w) `set-env` got deprecated.  As a new mechanism [Environment Files](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#environment-files) where introduced.